cdk/l3-cdk-resources/atlas-private-endpoint/README.md

```diff
@@ -0,0 +1,197 @@
+# @mongodbatlas-awscdk/atlas-basic-private-endpoint
+
+The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
+
+> This construct uses MongoDB [Level 1 construct](https://constructs.dev/search?q=&offset=0&tags=mongodb-published) and data structures for the [AWS CloudFormation Registry] Level 3 type.
+
+[L1 construct]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html
+[AWS CloudFormation Registry]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+## Description
+
+Creates a MongoDB Atlas Project, Cluster, DBuser, Private Endpoint and adds an IP entry to the IP Access List.
+
+
+## MongoDB Atlas API Docs
+
+For more information for each of the specific APIs used in this Level 3 AWS CDK resource please refer to: [Official Documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec)
+
+## Usage
+
+In order to use this library, you will need to activate this AWS CloudFormation Registry type in your account. You can do this via the AWS Management Console or using the [AWS CLI](https://aws.amazon.com/cli/) using the following command:
+
+```sh
+aws cloudformation activate-type \
+  --type-name MongoDB::Atlas::Cluster \
+  --publisher-id bb989456c78c398a858fef18f2ca1bfc1fbba082 \
+  --type RESOURCE \
+  --execution-role-arn ROLE-ARN
+  
+aws cloudformation activate-type \
+  --type-name MongoDB::Atlas::Project \
+  --publisher-id bb989456c78c398a858fef18f2ca1bfc1fbba082 \
+  --type RESOURCE \
+  --execution-role-arn ROLE-ARN
+
+aws cloudformation activate-type \
+  --type-name MongoDB::Atlas::DatabaseUser \
+  --publisher-id bb989456c78c398a858fef18f2ca1bfc1fbba082 \
+  --type RESOURCE \
+  --execution-role-arn ROLE-ARN  
+
+aws cloudformation activate-type \
+  --type-name MongoDB::Atlas::ProjectIpAccessList \
+  --publisher-id bb989456c78c398a858fef18f2ca1bfc1fbba082 \
+  --type RESOURCE \
+  --execution-role-arn ROLE-ARN  
+  
+aws cloudformation activate-type \
+  --type-name MongoDB::Atlas::PrivateEndpoint \
+  --publisher-id bb989456c78c398a858fef18f2ca1bfc1fbba082 \
+  --type RESOURCE \
+  --execution-role-arn ROLE-ARN
+```
+Alternatively:
```

Comment:
[nit] Could you add context on why the alternative command is required here? It's a little confusing

Reply:
I have removed it.